ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/scribble/v1/scribble/_cq_design_dialog/.content.xml

```diff
@@ -0,0 +1,80 @@
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Adaptive Form Scribble field"
+    sling:resourceType="cq/gui/components/authoring/dialog"
+    trackingFeature="core-components:design-dialog:adaptiveform-scribble:v1">
+    <content
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="granite/ui/components/coral/foundation/container">
+        <items jcr:primaryType="nt:unstructured">
+            <tabs
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/tabs"
+                    maximized="{Boolean}true">
+                <items jcr:primaryType="nt:unstructured">
+                    <formats
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Formats"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"
+                            margin="{Boolean}true">
+                        <items jcr:primaryType="nt:unstructured">
+                            <column
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/container">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <customFormats
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/multifield"
+                                        fieldDescription="The scribble formats that are allowed to be selected by an author."
+                                        fieldLabel="Custom Formats"
+                                        composite="true"
+                                        name="./allowedCustomFormats">
+                                        <field
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType = "granite/ui/components/coral/foundation/container"
+                                            name = "./allowedCustomFormats">
+                                            <items jcr:primaryType="nt:unstructured">
+                                                <customFormatKey
+                                                    jcr:primaryType="nt:unstructured"
+                                                    fieldDescription="The scribble format that will be displayed in formatter dropdown in dialog."
+                                                    fieldLabel="Custom Format Key"
+                                                    sling:resourceType = "granite/ui/components/coral/foundation/form/textfield"
+                                                    name = "customFormatKey"/>
```

Comment:
This is too vague, and its not clear what to add here. Can we add some samples here by default or improve the readme to provide  more details on this

Reply:
removed